_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -128,15 +128,15 @@ Some user agents treat the value of `aria-*` attribute as case-sensitive (even w
 
 #### Passed Example 9
 
-`aria-dropeffect` property with valid token list value
+This `div` element has an `aria-dropeffect` property with valid token list value.
 
 ```html
 <div role="dialog" aria-dropeffect="copy move"></div>
 ```
 
 #### Passed Example 10
 
-`aria-controls`, which is a required property for the role `scrollbar`, has `ID Reference list` that references at least one element existing in the same document tree.
+The second `div` element has an `aria-controls`, which is a required property for the role `scrollbar`, has `ID Reference list` that references at least one element existing in the same document tree.
```

Comment:
This example feels overly complicated due to the many other `aria-*` attributes in it. Could we achieve the same thing with less attributes? (i.e., make the example atomic testing only one thing)

Reply:
It is not a MUST to set other aria-*, except aria-controls, so I have updated accordingly.